main/Checkbox/index.tsx

```diff
@@ -84,15 +91,14 @@ const CheckboxGroup: FC<CheckboxGroupProps> = ({
   };
 
   return (
-    <Container>
+    <Container style={{flexWrap:"wrap"}}>
```

Comment:
For flexWrap styles, it would be more consistent to add to the StyledComponent side at the top.
(flexWrap 스타일의 경우 위쪽에 StyledComponent 쪽에 추가하는게 더 일관성이 있을 거 같아요.)

```js
const Container = styled.View`
 flex-direction: row;
 align-items: center;
 // like this
 flex-wrap: wrap
`;
```

Reply:
Thanks so much for your good review! I reflected the feedback you gave me.